src/components/modal/v2/parts/views/PayInFour/styles.scss

```
@@ -0,0 +1,320 @@
+@import '../../../styles/globals/mixins';
+
+.modal-wrapper.lander {
+    .header__wrapper {
+        .header__container {
+            background-color: transparent;
+    
+            .header__icons {
+                .pp-logo {
+                    img {
+                        height: 48px;
+                    }
+                }
+    
+                button {
+                    display: none;
+                }
+            }
+        }
+    
+        .header__content {
+            padding-bottom: 2rem;
+        }
+    }
+    
+    .content__background {
+        max-width: 933px;
+        margin-top: 0;
+    }
+    
+    .content__container {
+        background-color: transparent;
+    
+        .content__body {
+            border-top-left-radius: 100px;
+            border-bottom-right-radius: 100px;
+    
+            .dynamic__container {
+                overflow: hidden;
+                background-color: #fff;
+                border-top-left-radius: 100px;
+            }
+    
+            .content__row {
+                .donuts {
+                    .donuts__container {
+                        padding: 0 20px;
+                    }
+                }
+    
+                &.dynamic {
+                    max-width: 840px;
+                    margin: 0 auto;
+
+                    .branded-image {
+                        margin-top: 50px;
+                    }
+                }
+    
+                &.instructions {
+                    max-width: 315px;
+                }
+            }
+        }
+    }
+}
+
+.header__wrapper {
+    .header__container {
+        .header__icons {
+            button {
+                margin-top: 0.25rem;
+            }
+        }
+    }
+
+    .header__content {
+        h1 {
+            font-size: 1.75rem;
+            line-height: 36px;
+
+            @include desktop {
+                font-size: 2.75rem;
+                line-height: 48px;
+                margin: 0 auto;
+            }
+        }
+
+        h2 {
+            margin-top: 0.25rem;
+            font-size: 1rem;
+            line-height: 24px;
+            font-family: PayPalSansBig, Helvetica, Arial, Sans-Serif;
+
+            @include tablet {
+                margin-top: 1rem;
+            }
+
+            @include desktop {
+                font-size: 1.75rem;
+                line-height: 36px;
+                max-width: 577px;
+                margin: 1rem auto 0;
+            }
+         }
+    }
+}
```

```suggestion
    .header__content {
        h2 {
            @include mobile {
                padding: 20px 25px 25px;
                line-height: 24px;
            }
         }
    }
}
```
The header text on desktop was looking a bit big so after playing around with it I found that removing the styles here and replacing them with the above looks a bit closer to the mock. It relies more on the common styles now with some slight tweaks for short term.

---

updated